src/signals/IncidentMap/components/Header/styled.tsx

```diff
@@ -17,7 +17,7 @@ export const HeaderWrapper = styled.div`
   // z-index relative to map
   z-index: 1;
 
-  @media screen and ${breakpoint('max-width', 'tabletM')} {
+  @media screen and (${breakpoint('max-width', 'tabletM')}) {
```

Comment:
also you don't need the parentheses

Reply:
These parentheses remove this warning. 

![Screenshot 2023-02-06 at 11 46 17](https://user-images.githubusercontent.com/46756331/216952141-eaa51071-9bf4-4c0f-80cc-21005c21decc.png)

Reply:
That because of the plugin your'e using, my version doesnt. I guess we should remove them when they're not really needed.